demos/text-field.html

```diff
@@ -231,11 +230,21 @@ <h2>Text Field box</h2>
           <input id="box-required" type="checkbox">
           <label for="box-required">Required</label>
```

Comment:
While we're here, can we rename this label to "Required and must be at least 8 characters"?